### ISSUE TITLE
Segmonto property

### ISSUE THREAD

**Issue by gabays** [HTR-United/schema#14]
Would it be interesting to offer a _SegmOnto_ property?

**Comment by alix-tz** [comment]
Yes, this is a very good idea to add such information to the transcription guidelines !
